src/runtime/float.cpp

```diff
@@ -124,6 +125,8 @@ extern "C" Box* floatAdd(BoxedFloat* lhs, Box* rhs) {
         return floatAddFloat(lhs, static_cast<BoxedFloat*>(rhs));
     } else if (rhs->cls == long_cls) {
         return boxFloat(lhs->d + PyLong_AsDouble(rhs));
+    } else if (rhs->cls == complex_cls) {
+        return complexAdd(new BoxedComplex(lhs->d, 0), rhs);
```

Comment:
Do we need these?  Looks like cpython returns NotImplemented for `(1.0).__add__(1j)`.  Probably not a huge deal either way but was just curious if it meant that we are missing something somewhere else.